### PR TITLE
Event router e2e test

### PR DESCRIPTION
**Pull request by syedriko**
Migration of the event router e2e test from OAL

### CODE REVIEW

**Comment by jcantrill** [comment]
@syedriko please squash the commits if you are satisfied with the changes

File: docs/deploy-event-router.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Deploying Event Router
+The EventRouter is a simple service that listens to kubernetes evenets and writes them to `STDOUT`.  It is not managed by the `cluster-logging-operator`.
+Creating a deployment using the following template will enable event collection equivalent to the same capability from 3.11 releases or earlier.
+Currently, all events are collected by the cluster logging collector and indexed to the `infra` index when
```
Comment (jcantrill) [inline docs/deploy-event-router.md:4]: Please review the fluent.conf to see if this is a correct statement

Comment (syedriko) [reply docs/deploy-event-router.md:4]:
```
<rule>
  key CONTAINER_NAME
  pattern ^k8s_[^_]+_logging-eventrouter-[^_]+_
  tag kubernetes.journal.container._default_.kubernetes-event
</rule>

<match **_default_** **_kube-*_** **_openshift-*_** **_openshift_** journal.** system.var.log**>
  @type relabel
  @label @_LOGS_INFRA
</match>
```

Observed k8s events in Kibana's infra-000001 index pattern.

**Review by jcantrill — COMMENTED** [review]
/lgtm

**Pull request merged**